lib/ziti-tunnel-cbs/ziti_hosting.c

```diff
@@ -205,14 +213,45 @@ static ssize_t on_hosted_client_data(ziti_connection clt, uint8_t *data, ssize_t
     return len;
 }
 
-#define ZITI_MTU (15 * 1024)
 static void alloc_buffer(uv_handle_t *handle, size_t suggested_size, uv_buf_t *buf) {
-    *buf = uv_buf_init((char*) malloc(ZITI_MTU), ZITI_MTU);
+    struct hosted_io_ctx_s *io_ctx = handle->data;
+
+    if (io_ctx->in_wreqs < MAX_OUTSTANDING_WRITES) {
+        *buf = uv_buf_init((char*) malloc(ZITI_MTU), ZITI_MTU);
+    } else {
+        ZITI_LOG(VERBOSE, "max ziti writes[%d] reached for %s", io_ctx->in_wreqs, io_ctx->server_dial_str);
```

Comment:
this could destroy the disk if verbose logging is enabled and a mountain of data is trying to be sent. seems DOS-like to me. can it be throttled?

Reply:
this is on connection that ziti-edge-tunnel initiates, also this is no different from any other place we do verbose logging